.github/workflows/testdockertask.yml

```diff
@@ -0,0 +1,39 @@
+# Test docker-related tasks on linux & windows
+
+name: Test Dockertask
+
+on: [push, pull_request]
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  build:
+
+    runs-on: ${{ matrix.os }}
+    continue-on-error: true # until we fix tests on windows
```

Comment:
the same as for slurm - I'm not really convinced that there is any reason to use it. If I understand it correctly, whatever happens this will give us green light...